modules/host.go

```diff
@@ -36,7 +36,7 @@ type (
 		LockedStorageCollateral types.Currency `json:"lockedstoragecollateral"`
 		LostRevenue             types.Currency `json:"lostrevenue"`
 		LostStorageCollateral   types.Currency `json:"loststoragecollateral"`
-		PotentialStorageRevenue types.Currency `json:"potentialestoragerevenue"`
+		PotentialStorageRevenue types.Currency `json:"potentialstoragerevenue"`
```

Comment:
yeah... weird.
stepping through the commits of #1254, it looks like the typo was restored by a faulty rebase.
In 11a0e34c8a8484b558fe6117fa1083c3fdef08e4 *you* remove the typo, and in 12f298da4f45c60acf39876c711547899d4ef310, I add it back!